File: week_01/code_wars/thkwon/200826_even_or_odd.py
import unittest

EVEN = "Even"
ODD = "Odd"


def even_or_odd(number):
    return EVEN if number % 2 == 0 else ODD
    
    
class TestEvenOrOdd(unittest.TestCase):
    def test_should_return_even_when_number_is_2(self):
        self.assertEqual(even_or_odd(2), EVEN)

    def test_should_return_even_when_number_is_0(self):
        self.assertEqual(even_or_odd(0), EVEN)

    def test_should_return_odd_when_number_is_3(self):
        self.assertEqual(even_or_odd(3), ODD)

    def test_should_return_odd_when_number_is_negative_1(self):
        self.assertEqual(even_or_odd(-1), ODD)

File: week_01/code_wars/myeong/200826_EvenorOdd.cpp
using namespace std;
std::string even_or_odd(int number) {
  string integer_function;
  if (number < 0) {
    number = -number;
  }

  if (number % 2 == 1) {
    integer_function = "Odd";
  } else if (number % 2 == 0) {
    integer_function = "Even";
  }
  return integer_function;
}